src/app/myworkflows/my-workflow/my-workflow.component.html

```diff
@@ -152,7 +138,7 @@ <h3>Shared with me</h3>
                   {{ entryType$ | async }}, do one of the following:
                 </p>
                 <ul>
-                  <li *ngIf="entryType === EntryType.Service">
+                  <li *ngIf="entryType === EntryType.Service || entryType === EntryType.Notebook">
                     Press the 'Manage on GitHub' button and follow the GitHub apps web interface to create services from your GitHub
```

Comment:
This might be out of date because I don't see a Manage on GitHub button

Reply:
I remember thinking it didn't make sense, thanks for verifying. Updating to be as so, which I think makes sense:

```
<li>
  Press the 'Register a {{ entryType$ | async | titlecase }}' button 
  <span *ngIf="entryType === EntryType.Service || entryType === EntryType.Notebook; else manualRegistration">and follow the GitHub apps web interface to create {{ entryType$ | async }}s from your GitHub repositories.</span>
  <ng-template #manualRegistration >to manually add a {{ entryType$ | async }} using the registration wizard.</ng-template>
</li>
```
**For Services/Notebooks:**

Before:
![image](https://user-images.githubusercontent.com/97123241/233190911-81a62839-24eb-43fd-b337-b213e1e1e72b.png)

After:
![image](https://user-images.githubusercontent.com/97123241/233190996-4a82276c-931a-48fa-8d95-414acbb6a7e8.png)

**For workflows:**

Before:
![image](https://user-images.githubusercontent.com/97123241/233192987-4fe597de-a212-420a-924a-7f12fb7d12dd.png)

After:
![image](https://user-images.githubusercontent.com/97123241/233192420-ea9bcf61-1155-485b-bff3-4bfafc29d597.png)

Going to update `my-tool.component` to match while here:
Before:
![image](https://user-images.githubusercontent.com/97123241/233193657-ae71e19c-b89f-46d3-98bd-95de9f871c5a.png)
After:
![image](https://user-images.githubusercontent.com/97123241/233194959-80570c2d-f447-4d45-baed-2dc4a83e1c20.png)

Changed buttons to say "Register a {{ entryType }}" to match the other registration buttons:
![image](https://user-images.githubusercontent.com/97123241/233195089-66caff9a-4c82-4c2b-8465-242caa6ceaba.png)
![image](https://user-images.githubusercontent.com/97123241/233195128-ad80f966-e826-4464-a26b-b545a5dbb95b.png)